config/routes.rb

```diff
@@ -699,9 +700,12 @@
     namespace :alaveteli_pro do
       root to: 'dashboard#index', :as => :dashboard, :via => :get
       resources :draft_info_requests, :only => [:create, :update]
-      resources :info_requests, :only => [:new, :create, :update, :index] do
+      resources :info_requests, only: [:new, :create, :index] do
         get :preview, on: :new # /info_request/new/preview
       end
+      scope path: 'request/:url_title' do
```

Comment:
```suggestion
      scope path: 'info_requests/:url_title' do
```

Perhaps move this block down the file to the other Pro route scope.